test/integration/integration_test.go

```diff
@@ -239,14 +239,14 @@ func executingMinishiftCommandSucceedsOrFails(command, expectedResult string) er
 	if err != nil {
 		return err
 	}
-	success := (lastCommandOutput.ExitCode != 0 || len(lastCommandOutput.StdErr) != 0)
+	failure := (lastCommandOutput.ExitCode != 0 || len(lastCommandOutput.StdErr) != 0)
 	switch expectedResult {
 	case "succeeds":
-		if success {
+		if failure {
```

Comment:
@gbraad @LalatenduMohanty TBH I do not share the "confusing" feeling, so I am quite unsure how to resolve this problem, but I am trying it like this, does this look better to you?

```
commandFailed := (lastCommandOutput.ExitCode != 0 || len(lastCommandOutput.StdErr) != 0)
if (expectedResult == "succeeds" && commandFailed == true) {
    return fmt.Errorf("Command did not execute successfully. cmdExit: %d, cmdErr: %s", lastCommandOutput.ExitCode, lastCommandOutput.StdErr)
}
if (expectedResult == "fails" && commandFailed == false) {
    return fmt.Errorf("Command executed successfully, however was expected to fail. cmdExit: %d, cmdErr: %s", lastCommandOutput.ExitCode, lastCommandOutput.StdErr)
}
```
And the case where expectedResult is not "succeeds" nor "fails" I will cover in regex matching with group: `(succeeds|fails)`.